example/full/instance.properties

```diff
@@ -615,13 +684,65 @@ sleeper.default.rowgroup.size=8388608
 # The size of the pages in the Parquet files (default is 128KiB).
 sleeper.default.page.size=131072
 
-# The compression codec to use in the Parquet files
+# The compression codec to use in the Parquet files.
+# Valid values are: [uncompressed, snappy, gzip, lzo, brotli, lz4, zstd]
 sleeper.default.compression.codec=ZSTD
 
-# This specifies whether point in time recovery is turned on for DynamoDB tables. This default can
-# be overridden by a table property.
+# Whether dictionary encoding should be used for row key columns in the Parquet files.
+sleeper.default.parquet.dictionary.encoding.rowkey.fields=false
+
+# Whether dictionary encoding should be used for sort key columns in the Parquet files.
+sleeper.default.parquet.dictionary.encoding.sortkey.fields=false
+
+# Whether dictionary encoding should be used for value columns in the Parquet files.
+sleeper.default.parquet.dictionary.encoding.value.fields=false
+
+# This specifies whether point in time recovery is turned on for DynamoDB tables. This default can be
+# overridden by a table property.
 sleeper.default.table.dynamo.pointintimerecovery=false
 
 # This specifies whether queries and scans against DynamoDB tables used in the DynamoDB state store
-# are strongly consistent. This default can be overriden by a table property.
+# are strongly consistent. This default can be overridden by a table property.
 sleeper.default.table.dynamo.strongly.consistent.reads=false
+
+# Specifies the minimum number of leaf partitions that are needed to run a bulk import job. If this
+# minimum has not been reached, bulk import jobs will refuse to start
```

Comment:
Looks like we're missing a full stop at the end here

Reply:
Done, thanks